packages/mds-compliance-api/tsconfig.build.json

```diff
@@ -4,14 +4,16 @@
     "outDir": "dist"
   },
   "references": [
-    { "path": "../../packages/mds-api-server/tsconfig.build.json" },
-    { "path": "../../packages/mds-agency-cache/tsconfig.build.json" },
+    { "path": "../../packages/mds-compliance-service/tsconfig.build.json" },
     { "path": "../../packages/mds-db/tsconfig.build.json" },
-    { "path": "../../packages/mds-logger/tsconfig.build.json" },
+    { "path": "../../packages/mds-api-helpers/tsconfig.build.json" },
     { "path": "../../packages/mds-providers/tsconfig.build.json" },
-    { "path": "../../packages/mds-stream/tsconfig.build.json" },
-    { "path": "../../packages/mds-test-data/tsconfig.build.json" },
+    { "path": "../../packages/mds-api-server/tsconfig.build.json" },
+    { "path": "../../packages/mds-logger/tsconfig.build.json" },
+    { "path": "../../packages/mds-service-helpers/tsconfig.build.json" },
+    { "path": "../../packages/mds-logger/tsconfig.build.json" },
     { "path": "../../packages/mds-types/tsconfig.build.json" },
+    { "path": "../../packages/mds-schema-validators/tsconfig.build.json" },
     { "path": "../../packages/mds-utils/tsconfig.build.json" }
```

Comment:
It might be worth sorting these to make duplicates easier to spot. I see `mds-logger` at least is in here twice.